main.py

```diff
@@ -5,15 +5,35 @@
 import ffmpeg
 from ffmpeg import probe
 from ffprobe3 import FFProbe
+import re
 
 # Array of arrays with each internal array representing an expected input resolution.
 # Includes ratios called in confirmation of FFprobe stream request.
 # Gives shortform name for each expected corresponding .png watermark file.
 # .png files are sized the same as SD, HD, Full HD and cropped HD (1440x1080) video
-aspects = [[720,  "4:3",  "SD"],
-          [1280, "16:9",  "HD"],
-          [1920, "16:9", "FHD"],
-          [1440,  "4:3", "CHD"]]
+RESOLUTIONS = {
```

Comment:
In languages like Python that don't have constants (immutable variables), it's a convention to name static data in all caps.  
It doesn't actually make `RESOLUTIONS` into a constant - it's more of a signal to the coder that it's static data.

Reply:
Blimey. Thanks very much!  I need to read the style guide.

Reply:
Ah, probably not, unless you're really keen. But it's a useful reference.